localization/english/culture/wc_cultural_maa_innovations_l_english.yml

```
@@ -18,3 +19,186 @@
+ # Tribal Era #
+ ##############
+ 
+ innovation_archmage:0 "Archmages"
+ innovation_archmage_desc:0 "Weaving their intricate enchantments, these mages defend their people with all the magical powers at their disposal. The archmagi brandish magical blades and ancient staves that serve to channel their fierce energies in battle"
+
+ innovation_buccaneer:0 "Buccaneers"
+ innovation_buccaneer_desc:0 "The buccaneers are the unrivaled lords of the seas. They are highly skilled in sea combat, including, but not limited to, firearms and sailing."
+    
+ innovation_brute:0 "Brutes"
+ innovation_brute_desc:0 "Gnoll brutes are the largest and nastiest gnolls. They wear the best armor and use shields plucked from the bodies of their victims. Scraps of armor cover their thick yellow-green fur, and a wicked looking flail is held in one hand."
+    
+ innovation_candleholder:0 "Candleholders"
+ innovation_candleholder_desc:0 "You no take candle! We protect the Candle King."
+ 
+ innovation_dervishes:0 "Dervishes"
+ innovation_dervishes_desc:0 "The Wastewander Dervishes are the top fighters in the bandit ranks. Their scimitars are so sharp that they cut the wind around them."
+    
+ innovation_dire_troll:0 "Dire Trolls"
+ innovation_dire_troll_desc:0 "These trolls are warriors who have perfected their arts, traded portions of their lives for the dark gift of brutal strength. Dire trolls are short lived but make their presence on this world known to all."
+    
+ innovation_dragon:0 "Dragons"
+ innovation_dragon_desc:0 "Dragons are immense, powerful, winged reptilian creatures and they are the backbone of our strength."
+    
+ innovation_troll_ranged:0 "Troll Ranged Tactics"
+ innovation_troll_ranged_desc:0 "With our unique ranged armaments, we have produced regiments like the Headhunters."
+ 
+ innovation_hexer:0 "Hexers"
+ innovation_hexer_desc:0 "The extremely cruel and nefarious troll Hexers are extremely skilled at manipulating the chemical processes in friendly warriors to increase their fighting abilities. A few of them know ancient rituals to temporarily break or even take over the wills of enemy soldiers."
+
+ innovation_kodo:0 "Kodo Taming"
+ innovation_kodo_desc:0 "Tauren legends abound regarding the great horned kodos of Kalimdor’s Barrens. Some myths tell of rare kodos that are bound to the spirits of the sky and storm, thus allowing the colossal beasts to harness the powers of lightning itself. While kodos are generally considered gentle by nature, they are fierce opponents when threatened and have occasionally been known to devour their enemies whole. Our tamers have successfully learned how to communicate and tame these grand creatures."
+ 
+ innovation_marauder:0 "Marauders"
+ innovation_marauder_desc:0 "Like most of thier kin, Marauders are savagery and brutality incarnate.  Marauders are fast, aggresive and show no mercy. Dawning the hides of their enemies Marauders are not for taking prisoners."
+    
+ innovation_magi:0 "Magis"
```

Magi is a plural form of mage (https://wowpedia.fandom.com/wiki/Ogre_mage)
```suggestion
 innovation_magi:0 "Magi"
```